backend/tournesol/views/ratings.py

```diff
@@ -25,7 +25,7 @@
             | Q(user__comparisons__video_2=F("video"))
         ),
     )
-).order_by("-pk")
+).order_by("-video__publication_date", "-pk")
```

Comment:
An explicit order is necessary to get a deterministic pagination, otherwise the order may change after refresh or after selecting a filter. The publication date looks more natural but `pk` is still useful as multiple videos may have the same publication date (no time is currently stored).